user-manual/appraisal/appraisal.rst

```diff
@@ -290,27 +296,47 @@ edited or deleted and further acts can be added.
 Adding Files to ArchivesSpace Resources and Starting SIPs
 ---------------------------------------------------------
 
-Files from transfers in the Backlog pane can be associated with ArchivesSpace
-resources by dragging files from the backlog to digital object components.
+Transfer files in backlog can easily be associated with ArchivesSpace resources by dragging and dropping files from the backlog to digital object components.
```

Comment:
Question: Should backlog remain capitalised as a proper noun from AM's perspective? i.e. the 'Backlog' tab.

Reply:
I think there's the Backlog tab/pane, which should be capitalized, and then the backlog as a concept, which could remain uncapitalized if the writer wishes. In this case, the use of the small-b backlog could refer to either - perhaps it would be more clear/consistent if this read "dragging and dropping files from the Backlog pane to digital object components in the ArchivesSpace pane"

Reply:
No prob- I will update to read "dragging and dropping files from the Backlog pane to digital object components in the ArchivesSpace pane". Thanks you two.